doc/explanation/networks.md

```diff
@@ -49,7 +49,11 @@ Fully controlled networks create network interfaces and provide most functionali
 LXD supports the following network types:
 
 {ref}`network-bridge`
-: A network bridge creates a virtual L2 ethernet switch that instance NICs can connect to, making it possible for them to communicate with each other and the host.
+: % Include content from [../reference/network_bridge.md](../reference/network_bridge.md)
+  ```{include} ../reference/network_bridge.md
+      :start-after: <!-- Include start bridge intro -->
+      :end-before: <!-- Include end bridge intro -->
+  ```
```

Comment:
Make one commit which has those kind of changes (removes text replaces by matching include)